Adds documentation about combined HA and Authentication uses (#9426)
confluentinc/ksql: docs/operate-and-deploy/high-availability.md

forwarded to servers whose current offset is more than `10,000` positions
behind the end offset of the changelog topic are rejected.

## Compatability with Authentication

> `authentication.skip.paths` should be set with both `/lag` and `/heartbeat`.

Reviewer: ```suggestion
Set the `authentication.skip.paths` config with both `/lag` and `/heartbeat`.
```

> This would allow ksqlDB cluster instances to communicate without authenticating between each other.

Reviewer: ```suggestion
This enables ksqlDB cluster instances to communicate without authenticating between each other.
```

> The configuration prevents the error below.

Reviewer: ```suggestion
This configuration prevents the following error.
```

```
ksqldb-server1     | [...] ERROR Failed to handle request 401 /heartbeat (io.confluent.ksql.api.server.FailureHandler:38)
ksqldb-server1     | io.confluent.ksql.api.server.KsqlApiException: Unauthorized
```

> It's recommended to block the traffic from outside your cluster to those endpoints for security reasons.

Reviewer: ```suggestion
For security reasons, this configuration is recommended to block the traffic from outside your cluster to those endpoints.
```
